src/processes/scoring/update-user-score-on-daily-process-phase1.js

```diff
@@ -54,6 +54,10 @@ const updateUserScoreOnDailyProcessPhase1 = async (
     userDoc.last_daily_process = processTime;
     userDoc.updated_at = moment().utc().format(REGULAR_TIME_FORMAT);
 
+    // set user anonymous
+    const currentUser = await UsersFunction.getUserFollowerList(userDoc._id);
+    userDoc.is_anonymous = currentUser.is_anonymous;
```

Comment:
mas @n-permana I'm not sure this `currentUser` returns an user object. Rather it should returns an array. 
So accessing `is_anonymous` will always return null

Reply:
This is getUserFolllowerList function.

![image](https://github.com/BetterSocial/bettersocial-dev-queue/assets/55779793/585621c2-e3ad-42f8-acad-4cb0962db787)

Reply:
sorry mas, salah panggil function. sudah saya update.